model/components/prometheus.yml

```diff
@@ -0,0 +1,105 @@
+---
+components:
+
+  # prometheus
+  - id: prometheus
+    name: Prometheus
+    description: Prometheus is an open-source systems monitoring and alerting toolkit.
+    link: https://prometheus.io/
+    logo: prometheus
+    categories:
+      - agent
+    capabilities:
+      - tech:
+        - java
+        - go
+        - scala
+        - python
+        - ruby
+        - bash
+        - c++
+        - common-lisp
+        - elixir
+        - erlang
+        - haskell
+        - lua
+        - .net
+        - c#
+        - nodejs
+        - php
+        - rust
+      - aspects:
+        - metrics-app
+        - metrics-process
+
+  # prometheus Server
+  - id: prometheus-server
+    name: Prometheus Server
+    description: The Jaeger collector receives traces from Jaeger agents and runs them through a processing pipeline. Currently our pipeline validates traces, indexes them, performs any transformations, and finally stores them.
+    link: https://prometheus.io/
+    logo: prometheus
+    categories:
+      - collector
+      - visualization
+      - storage
+    connections:
+      - graphite
+      - influx-db
+      - chronix
+      - weavecortex
+      - crate-db
+      - open-tsdb
+      - postgresql
+      - timescala-db
+      - grafana
+      - prometheus-exporter
+      - prometheus
+
+  # prometheus alerting
+  - id: prometheus-alerting
+    name: Prometheus Alert Manager
+    description: The Alertmanager handles alerts sent by client applications such as the Prometheus server. It takes care of deduplicating, grouping, and routing them to the correct receiver integration such as email, PagerDuty, or OpsGenie. It also takes care of silencing and inhibition of alerts.
+    link: https://prometheus.io/
+    logo: prometheus
+    categories:
+      - alerting
+    connections:
+      - prometheus-server
+      - pager-duty
+      - email
+      - slack
+      - hip-chat
+
+  # prometheus exporter
+  - id: prometheus-exporter
+    name: Prometheus Exporter
+    description: Libraries and servers which help in exporting existing metrics from third-party systems as Prometheus metrics.
+    link: https://prometheus.io/
+    logo: prometheus
+    categories:
+      - agent
+      - exporter
+    capabilities:
+      - aspects:
+        - metrics-app
+        - metrics-process
+        - metrics-system
+      - frameworks:
+        - ha-proxy
+        - aws-cloud-watch
+        - collectd
+        - jmx
+        - snmp
+        - statsd
+        - blackbox
+        - node-system-metrics
+        - graphite
+        - influx-db
+      - os:
+        - linux
```

Comment:
also, please one commit.. You can name commit message `closes #9: added Prom...", thus way issue #9 is automatically closed when we merge this and the issue and MR are referenced. OK?

Reply:
yes sure I just wanted to wait until the review so I don't have to squash it several times